### PR TITLE
Document more client errors

### PR DESCRIPTION
**Pull request by matr1x-hackmud**
### Problem

Client errors guide needs more common client errors

### Context

We need to add the ban appeal process. I'm also seeking feedback on any other known/common client bugs

### CODE REVIEW

File: docs/guides/misc/client_errors.mdx [hunk 1]
```diff
@@ -4,6 +4,51 @@ title: Known Client Errors
...
+
+Your account is banned from hackmud. A reason and expiration time is given in the error message.
+
+To appeal your ban, ((%cappeal instructions go here%))
+
+## 403 Forbidden
+
+### Timestamp validation failed
+
+hackmud requires the operating system's clock to be properly set to its closest timezone. Check that your system clock is congruent with https://time.is
```
Comment (seanmakesgames) [inline docs/guides/misc/client_errors.mdx:36]:
```suggestion
hackmud requires the operating system's clock to be properly set to its closest timezone. Check that your system clock is synchronized with https://time.is
```

File: docs/guides/misc/client_errors.mdx [hunk 2]
```diff
@@ -4,6 +4,51 @@ title: Known Client Errors
...
+
+## no client output
+
+If there is no client output after launching the game, this is typically indicative of a man-in-the-middle (such as a censoring ISP) blocking the connection to hackmud.
+
+Please check that your web browser can access https://hackmud.com; for more guidance, please open a support ticket on [the discord](https://discord.gg/QRTXHaz).
+
+## 503 Maintenance
+
+hackmud is offline for planned maintenance. Please visit [the discord](https://discord.gg/QRTXHaz), Steam announcements or [Twitter](https://x.com/hackmudgame) for updates.
```
Comment (seanmakesgames) [inline docs/guides/misc/client_errors.mdx:46]:
```suggestion
hackmud is offline for planned maintenance. Please visit [the discord](https://discord.gg/QRTXHaz) or [Twitter](https://x.com/hackmudgame) for updates.
```

We don't really use steam announcements for update notifications.

**Review by seanmakesgames — COMMENTED** [review]
Thanks for putting this together. It's great and should help with common connection issue stuff.

**Comment by matr1x-hackmud** [comment]
Sorry for the double email

File: docs/guides/misc/client_errors.mdx [hunk 3]
```diff
@@ -4,6 +4,51 @@ title: Known Client Errors
...
+- Steam being closed on your operating system
+- Launching the hackmud exectuable directly instead of through steam
+
+Make sure Steam & your OS are up-to-date (might need a retart) and that you have a stable connection to Steam
+
+### Your account has been banned
+
+Your account is banned from hackmud. A reason and expiration time is given in the error message.
+
+To appeal your ban, ((%cappeal instructions go here%))
```
[the remark this answers is not in the document]
Comment (matr1x-hackmud) [reply docs/guides/misc/client_errors.mdx]: I've ported over all the relevant info in the most recent commit

**Review by hmdunce — APPROVED** [review]

**Pull request merged**